source/auth/tests/connection-string.yml

```diff
@@ -1,367 +1,333 @@
 tests:
     -
         description: "should use the default source and mechanism"
-        uri: "mongodb://user:password@localhost"
-        hosts: ~
+        uri: "mongosource://user:password@localhost"
```

Comment:
What is the reason for changing the scheme to `mongosource` in these tests? AFAIK only `mongodb` and `mongodb+srv` are supported so I would expect this to result in an invalid URI.

Was this a side effect of replacing `db:` with `source:` in the `credential` object?

Reply:
😂 Yes!  Will fix.